Kickstarter-iOS/Views/Storyboards/Search.storyboard

```diff
@@ -180,97 +205,122 @@
+                                        <constraint firstItem="keN-0e-evx" firstAttribute="top" secondItem="LC3-rr-wed" secondAttribute="top" constant="6" id="zvd-5r-4Fe"/>
                                     </constraints>
                                     <edgeInsets key="layoutMargins" top="24" left="16" bottom="24" right="16"/>
                                 </tableViewCellContentView>
                                 <connections>
-                                    <outlet property="columnsStackView" destination="7Bu-Q1-vRs" id="KFD-Fm-ZwH"/>
-                                    <outlet property="dateStackView" destination="l2r-iG-wDQ" id="iNr-wd-Ny8"/>
-                                    <outlet property="deadlineSubtitleLabel" destination="z2v-lu-hrL" id="SbE-Qf-de7"/>
-                                    <outlet property="deadlineTitleLabel" destination="cAv-H2-A4X" id="Vj0-Od-zbI"/>
-                                    <outlet property="fundingTitleLabel" destination="dYI-q3-lWp" id="XLp-wi-fZ0"/>
-                                    <outlet property="imageShadowView" destination="3Ie-p2-cOs" id="BU8-BU-Wek"/>
-                                    <outlet property="projectImageView" destination="o5j-hv-hBJ" id="2O0-NT-KAj"/>
-                                    <outlet property="projectImageWidthConstraint" destination="aY4-mQ-cym" id="nQ1-1w-hMB"/>
-                                    <outlet property="projectLabel" destination="kqx-C7-H1c" id="pgZ-tt-V2b"/>
-                                    <outlet property="projectNameContainerView" destination="bmM-bP-tEX" id="kxy-iy-JCr"/>
-                                    <outlet property="separateView" destination="TX0-Jd-jBC" id="04v-VK-gDm"/>
-                                    <outlet property="statsStackView" destination="a95-YE-evX" id="tTO-Cr-5is"/>
+                                    <outlet property="cardView" destination="hna-1r-IBt" id="cDG-bc-Si8"/>
+                                    <outlet property="metadataBackgroundView" destination="LC3-rr-wed" id="bcc-Yq-xXW"/>
+                                    <outlet property="metadataIconImageView" destination="d6r-M1-e1G" id="GJD-jz-SpE"/>
+                                    <outlet property="metadataLabel" destination="2Vq-Ze-Jnt" id="hR7-8W-OB5"/>
+                                    <outlet property="percentFundedLabel" destination="n2e-oy-cd3" id="bds-d3-G0B"/>
+                                    <outlet property="progressBarView" destination="nzr-VY-8lM" id="uVM-4n-HHt"/>
+                                    <outlet property="progressStaticView" destination="ZM0-2K-7i3" id="MVx-x6-cNj"/>
+                                    <outlet property="projectImageView" destination="jOZ-gh-bkN" id="Jy2-XA-mxq"/>
+                                    <outlet property="projectLabel" destination="Z2K-XW-AnR" id="OZd-P5-G64"/>
```

Comment:
It looks like the `SearchProjectCell` is still in the Storyboard. Can you remove it?